schema/transportation/segment.yaml

```diff
@@ -62,6 +63,78 @@ properties:
       routes: { "$ref": "#/$defs/propertyDefinitions/routes" }
 "$defs":
   propertyDefinitions:
+    destinationLabelType:
+      description: >-
+        The type of object of the destination label.
+      type: string
+      enum: 
+      - street
+      - country 
+      - route_ref
+      - toward_route_ref
+      - unknown
+    destinations:
+      description: >-
+        Describes objects that can be reached by following a transportation segment in the
+        same way those objects are described on signposts or ground writing that a traveller
+        following the segment in would observe in the real world. This allows navigation systems
+        to refer to signs and observable writing that a traveller actually sees.
+      type: array
+      items:
+        type: object
+        unevaluatedProperties: false
+        required: [labels, from_connector_id, to_connector_id, to_segment_id, final_heading]
+        properties:
+          labels:
+            description: >-
+              Labeled destinations that can be reached by following the segment.
+            type: array
+            items:
+              type: object
+              unevaluatedProperties: false
+              required: [value, type]
+              properties:
+                value:
+                  description: >-
+                    Names the object that is reached
+                  type: string
+                  pattern: ^(\S.*)?\S$
+                type: { "$ref": "#/$defs/propertyDefinitions/destinationLabelType" }
+            minItems: 1
+            uniqueItems: true
+          symbols:
+            description: >-
+              A collection of symbols or icons present on the sign next to current 
+              destination label.
+            type: array
+            items: { "$ref": "#/$defs/propertyDefinitions/destinationSignSymbol" }
+            uniqueItems: true
+            minLength: 1            
+          from_connector_id:          
+            description: >-
+              Identifies the point of physical connection on this segment before which
+              the destination sign or marking is visible.
+            type: string
+          to_segment_id:
+            description: >-
+              Identifies the segment to transition to reach the destination(s) labeled 
+              on the sign or marking.
+            type: string
```

Comment:
Do we have an ID pattern to validate these?

Reply:
no, we don't...schema has multiple fields that represent id references, all of them currently only have `type: string` constraint